static/app/components/modals/inviteMissingMembersModal/index.tsx

```diff
@@ -45,45 +47,52 @@ export function InviteMissingMembersModal({
     role: organization.defaultRole,
     teamSlugs: new Set<string>(),
     externalId: member.externalId,
-    selected: true,
+    selected: false,
   }));
+  const pageCount = Math.floor(initialMemberInvites.length / ROWS_PER_PAGE);
   const [memberInvites, setMemberInvites] =
     useState<MissingMemberInvite[]>(initialMemberInvites);
   const referrer = missingMembers.integration + '_nudge_invite';
   const [inviteStatus, setInviteStatus] = useState<InviteStatus>({});
   const [sendingInvites, setSendingInvites] = useState(false);
   const [complete, setComplete] = useState(false);
+  const [currentPage, setCurrentPage] = useState<number>(0);
 
   const api = useApi();
 
   if (memberInvites.length === 0 || !organization.access.includes('org:write')) {
     return null;
   }
 
-  const setRole = (role: string, index: number) => {
+  const setRole = (role: string, email: string) => {
```

Comment:
these use email instead of index now because the full list of missing members is sliced. also, email is still a unique identifier